server/tests/api/videos/videos-common-filters.ts

```diff
@@ -162,17 +176,21 @@ describe('Test videos filter', function () {
       return res.body.data as Video[]
     }
 
-    async function getVideosNames (options: {
-      server: PeerTubeServer
-      isLocal?: boolean
-      include?: VideoInclude
-      privacyOneOf?: VideoPrivacy[]
-      token?: string
-      expectedStatus?: HttpStatusCode
-    }) {
+    async function getVideosNames (
+      options: {
+        server: PeerTubeServer
+        isLocal?: boolean
+        include?: VideoInclude
+        privacyOneOf?: VideoPrivacy[]
+        token?: string
+        expectedStatus?: HttpStatusCode
+      },
+      skipSubscription = false
```

Comment:
Why not put `skipSubscription` in `options`?

Reply:
Correct in https://github.com/Chocobozzz/PeerTube/pull/5665/commits/cee2a3fc25413173a465eb2392681605261cff78.